assemblies/assembly-installing-kie-server-on-wls.adoc

```diff
@@ -26,9 +26,9 @@ include::{enterprise-dir}/weblogic/wls-was-jms-configure-con.adoc[leveloffset=+2
 include::{enterprise-dir}/weblogic/wls-jms-create-proc.adoc[leveloffset=+3]
 include::{enterprise-dir}/weblogic/wls-jms-create-module-proc.adoc[leveloffset=+3]
 include::{enterprise-dir}/weblogic/wls-jms-factory-create-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/installation/jms-factories-ref.adoc[leveloffset=+4]
+//include::{enterprise-dir}/installation/jms-factories-ref.adoc[leveloffset=+4]
```

Comment:
Shouldn't those docs be rather deleted than commented out?

Reply:
Sure, I removed.

Reply:
@ksuta do you have any other comments or doc you approve the other changes?

Reply:
Sorry @ksuta I see your comment in the jira. Thank you!